scripts/create_release.py

```diff
@@ -14,8 +14,10 @@
 import emsdk  # noqa
 
 
-def version_to_list(version_string):
-  return [int(part) for part in re.split('[.-]', version_string)[:3]]
+def version_key(version_string):
+  parts = re.split('[.-]', version_string)
+  key = [[int(part) for part in parts[:3]], -len(parts), parts[3:]]
```

Comment:
so this comparison is basically X.Y.Z-a < X.Y.Z-b < X.Y.Z ?

Reply:
Yup, exactly.